packages/fuels-contract/src/contract.rs

```diff
@@ -182,9 +182,19 @@ impl Contract {
     ) -> Result<ContractId, Error> {
         let (mut tx, contract_id) =
             Self::contract_deployment_transaction(compiled_contract, wallet, params).await?;
+
+        let client = &wallet
+            .get_provider()
+            .expect("Can't get wallet provider?")
```

Comment:
`Can't get wallet provider?` I don't know, you tell me 😂
```suggestion
            .expect("Can't get wallet provider")
```

Reply:
:smile: